modules/nw-banp-audit-logging-concept.adoc

```diff
@@ -0,0 +1,65 @@
+// Module included in the following assemblies:
+//
+// * networking/openshift_network_security/AdminNetworkPolicy/logging-anp-policy.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="nw-banp-audit-logging-concept_{context}"]
+= BaselineAdminNetworkPolicy audit logging
+
+Audit logging is enabled in the `BaselineAdminNetworkPolicy` CR by annotating an ANP policy with the `k8s.ovn.org/acl-logging` key such as in the following example:
+
+.Example of annotation for `BaselineAdminNetworkPolicy` CR
+[%collapsible]
+====
+[source,yaml]
+----
+apiVersion: policy.networking.k8s.io/v1alpha1
+kind: BaselineAdminNetworkPolicy
+metadata:
+  annotations:
+    k8s.ovn.org/acl-logging: '{ "deny": "alert", "allow": "alert"}'
+  name: default
+spec:
+  subject:
+    namespaces:
+      matchLabels:
+          tenant: red
+  ingress:
+  - name: "default-allow-ns"
+    action: "Allow"
+    from:
+    - namespaces:
+          matchLabels:
+            tenant: blue
+  - name: "default-deny-ns"
+    action: "Deny"
+    from:
+    - namespaces:
+          matchLabels:
+            tenant: orange
+----
+====
+
+In the example, an event in which any of the namespaces with the label `tenant: blue` are allowed ingress to any of the namespaces with the label `tenant: red`, a log is generated.
+
+.Example ACL allow log entry for an `BaselineAdminNetworkPolicy` CR
+[source,text]
+----
+2024-05-28T20:39:07.131Z|00004|acl_log(ovn_pinctrl0)|INFO|name="BANP:default:Ingress:0", verdict=allow, severity=alert, direction=to-lport: tcp,vlan_tci=0x0000,dl_src=0a:58:0a:83:00:01,dl_dst=0a:58:0a:83:00:0e,nw_src=10.128.2.18,nw_dst=10.131.0.14,nw_tos=0,nw_ecn=0,nw_ttl=62,nw_frag=no,tp_src=42984,tp_dst=8080,tcp_flags=syn
+2024-05-28T20:42:06.270Z|00011|acl_log(ovn_pinctrl0)|INFO|name="BANP:default:Ingress:1", verdict=drop, severity=alert, direction=to-lport: tcp,vlan_tci=0x0000,dl_src=0a:58:0a:83:00:10,dl_dst=0a:58:0a:83:00:0e,nw_src=10.131.0.16,nw_dst=10.131.0.14,nw_tos=0,nw_ecn=0,nw_ttl=64,nw_frag=no,tp_src=46866,tp_dst=8080,tcp_flags=syn
+----
+
+The following table describes BANP annotation values:
```

Comment:
Suggestion: same comment here, where I'm not sure of the value of the table if it only has a single annotation in it. Up to you.